assets/icon-small.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is the difference to `assets/icon.svg`? They have the same file size.

Reply:
The height and width here are different.  The autocomplete icon uses a 128 where as the default is 256

Reply:
Given that these are svg files, are the dimensions important? Can the Mattermost server scale the image?

Reply:
testing in my local, if I remove the hard-coded values in the original svg it appears to work for all scenarios.  I'll go with that approach instead